pkg/sessions/redis/redis_store_test.go

```diff
@@ -1,6 +1,9 @@
-package redis
+package redis_test
```

Comment:
Is there a benefit to making this part of the test package? I never really understood why this helps, I assume the compiler ignores all the test code anyway?

Reply:
I'm gonna revisit the legacy_helper -- I'm thinking trying to consolidate the legacy test cases into 1 area would help DRY up the code... but it caused more headaches than I think its worth for tests we'll likely fully deprecate in V7 and rip out.

It caused circular imports -- which made me need to change the package of the test code that consumed them to `*_test` to avoid the circular import compile error.